docs/apis/inappbrowser.md

```diff
@@ -0,0 +1,330 @@
+---
+title: InAppBrowser Capacitor Plugin API
+description: The InAppBrowser Plugin provides a web browser view that allows you to load any web page externally. It behaves as a standard web browser and is useful to load untrusted content without risking your application's security.
+custom_edit_url: https://github.com/ionic-team/capacitor-os-inappbrowser/blob/main/README.md
+editApiUrl: https://github.com/ionic-team/capacitor-os-inappbrowser/blob/main/src/definitions.ts
+sidebar_label: InAppBrowser
+---
+
+# @capacitor/inappbrowser
+
+The InAppBrowser Plugin provides a web browser view that allows you to load any web page externally. It behaves as a standard web browser and is useful to load untrusted content without risking your application's security.
+
+## Install
+
+```bash
+npm install @capacitor/inappbrowser
+npx cap sync
+```
+
+## Supported Platforms
+
+- iOS
+- Android
+
+#### Android
+
+The InAppBrowser plugin requires a minimum Android SDK target of 26. This is higher than the default that comes with your Capacitor application. You can update this value in your `android/variables.gradle` file.
+
+```gradle
+ext {
+    minSdkVersion = 26
+}
+```
+
+You will need to modify the `allprojects > repositories` section in your `android/build.gradle` file to include the Outsystems repository. Your `android/build.gradle` file should look similar to this after adding the repository.
+
+```gradle
+allprojects {
+    repositories {
+        google()
+        mavenCentral()
+        maven {
+            url 'https://pkgs.dev.azure.com/OutSystemsRD/9e79bc5b-69b2-4476-9ca5-d67594972a52/_packaging/PublicArtifactRepository/maven/v1'
+            name 'Azure'
+            credentials {
+                username = "optional"
+                password = ""
+            }
+            content {
+                includeGroup "com.github.outsystems"
+            }
+        }
+    }
+}
+```
+
```

Comment:
Can we add an example section here showing how to use this in an application? I'm sure it's really simple, but it's some feedback I hear quite often about the plugin docs that they lack examples.